test/optimus/4kpbmc/test_inputs.json

```diff
@@ -0,0 +1,26 @@
+{
+  "TestOptimusPR.expected_bam_hash": "83234b3b714c96d897b44d91169933bd",
+  "TestOptimusPR.reference_matrix": "gs://hca-dcp-sc-pipelines-test-data/optimus-test-data/referenceMatrix.rds",
+  "TestOptimusPR.expected_cell_metric_hash": "eb56bfec00cb35ea176c00cebc44c4e3",
```

Comment:
need to make sure these checksums are correct, and/or remove them and make sure that the validation doesn't require them.

Reply:
fixed